backend/src/constants/ErrorMessage.ts

```diff
@@ -7,4 +7,7 @@ export default {
     SOMETHING_WENT_WRONG: 'Something went wrong',
     INVALID_MODE: 'Invalid mode selected',
     DUPLICATE_RECORD: 'Duplicate record found',
+    QUOTA_EXCEEDED: 'Quota exceeded for this month',
+    RECORD_NOT_FOUND: 'Api Key provided not found',
+    API_KEY_VALIDATION_FAILED: 'Api Key is not in the right format as described',
```

Comment:
Sorry just so i understand the context, as described where?

Reply:
in readme I will add that